### PR TITLE
Move team before giver/recipient for feedback result responses #4672

### LINKED ISSUE #4672: Move team before giver/recipient for feedback result responses

### PR DESCRIPTION
**Pull request by unyoungwax**
Fixes #4672 

In question view, the team column is moved before the giver/recipient column.
screenshot_q

In giver > question > recipient view, the team column is between the photo column and the recipient column.
screenshot_gqr

In recipient > question > giver view, the team column is between the photo column and the giver column.
screenshot_rqg

### CODE REVIEW

File: src/test/resources/pages/instructorFeedbackResultsPageOpen.html [hunk 1]
```diff
@@ -2909,7 +2909,7 @@
                         <a href="/index.html">
                            TEAMMATES
                         </a>
-                        V${version}]
+                        V4.18]
```
Comment (LowWeiLin) [inline src/test/resources/pages/instructorFeedbackResultsPageOpen.html]: This should not be changed

**Comment by LowWeiLin** [comment]
Code looks correct, just need to fix the tests.

File: src/test/resources/pages/instructorFeedbackResultsPageOpen.html (hunk 1 above)
Comment (thyageshm) [inline src/test/resources/pages/instructorFeedbackResultsPageOpen.html]: Was this done using Godmode?

Comment (LowWeiLin) [inline src/test/resources/pages/instructorFeedbackResultsPageOpen.html]: No

**Comment by thyageshm** [comment]
@unyoungwax would you be able to wrap this up today?

**Comment by unyoungwax** [comment]
The related tests are fixed.

**Pull request merged**